app/scripts/metamask-controller.js

```diff
@@ -257,7 +257,7 @@ export default class MetamaskController extends EventEmitter {
           const metamaskState = await this.getState()
           backEndMetaMetricsEvent(metamaskState, {
             customVariables: {
-              errorMessage: txMeta.simulationFails.reason,
+              errorMessage: txMeta.simulationFails?.reason || 'Unknown Reason',
```

Comment:
```suggestion
              errorMessage: txMeta.simulationFails?.reason,
```
Is the `Unknown Reason` part required? Falling back to `null` seems perfectly fine - that's what was intended originally I suspect.